pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[project]
+name = "TF2.4_IVIM-MRI-CodeCollection"
+version = "0.1.0"
+description = "IVIM MRI signal processing and NIfTI/DICOM processing tools"
+requires-python = ">=3.9"
+readme = "README.md"
+dependencies = [
```

Comment:
I'm noticing these dependencies don't completely match with those in requirements.txt. Is that intentional?